### DISCUSSION TITLE
partition_html is not returning the content of an <img> tag within a <td> tag

### DISCUSSION THREAD

**Discussion by samiislam** [Unstructured-IO/unstructured#3664] [Q&A]
Hi,

I am trying to use partition_html to return a Table element with the following similar format:

```
<tbody><tr>
<td><a href="/wiki/index.php/File:Header.png" class="image"><img alt="Header.png" src="/wiki/images/2/7/Header.png" decoding="async" width="20" height="211" /></a>
</td>
```

I would like to obtain the information of the `<img>` tag, particularly the name of the src file but partition_html is not returning it.

What would be the best way to obtain it?
Thanks!
Sami

**Comment by jackey90** [comment]
Does anyone konw how to solve this ?
